packages/wonder-blocks-birthday-picker/src/components/__docs__/birthday-picker.stories.js

```diff
@@ -12,7 +12,7 @@ import ArgTypes from "./birthday-picker.argtypes.js";
 import BirthdayPicker from "../birthday-picker.js";
 
 export default {
-    title: "BirthdayPicker",
+    title: "BirthdayPicker / BirthdayPicker",
```

Comment:
Tweak to make the docs look consistent.

Reply:
Oh ok. It looks really odd as it stands. Given that we have to consolidate with Mobile WB for a proper Wonder Blocks design system site, I had thought this would remain engineering docs.

I'll switch it back.